onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1016,6 +1016,9 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain,
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 13, BFloat16, ReduceSum);
 #endif
 
+// OpSet 14
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCudaExecutionProvider, kOnnxDomain, 14, CumSum);
```

Comment:
don't we need to consume the onnx change bumping up the version of this operator ?

Reply:
Update: I saw that we are consuming the onnx commit pertaining to this change